app/manager/discography/page.jsx

```
@@ -2,3 +1,53 @@
+      <div className="relative group">
+        <button className="md-fab bg-[--md-sys-color-primary] text-[--md-sys-color-on-primary] hover:bg-[--md-sys-color-primary-container] hover:text-[--md-sys-color-on-primary-container] active:bg-[--md-sys-color-primary-container] active:text-[--md-sys-color-on-primary-container] shadow-md p-2 rounded-full flex items-center justify-center gap-2 w-12 h-12 min-w-[48px] min-h-[48px]" aria-label="Toggle add menu">
+          <md-ripple />
+          <span className="material-symbols-outlined group-hover:rotate-45 transition-transform">add</span>
+        </button>
+
+        <div className="absolute bottom-full right-0 mb-2 flex flex-col gap-2 opacity-0 invisible group-hover:opacity-100 group-hover:visible transition-all">
+          <button
+            onClick={() => router.push('/manager/discography/add?type=album')}
+            className="md-fab bg-[--md-sys-color-primary] text-[--md-sys-color-on-primary] hover:bg-[--md-sys-color-primary-container] hover:text-[--md-sys-color-on-primary-container] shadow-md p-2 rounded-full flex items-center justify-center gap-2 relative group/button w-12 h-12 min-w-[48px] min-h-[48px]"
+            aria-label="Add new album"
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)